### PR TITLE
Add new links for operate first open data hub

### PR DESCRIPTION
**Pull request by Shreyanand**
## Related Issues and Dependencies
[Issue 101](https://github.com/operate-first/operate-first.github.io/issues/101) 

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

In this PR, I add link to project image on public JH, MOC.

### CODE REVIEW

File: docs/blog/blog.md [hunk 1]
```diff
@@ -212,10 +212,10 @@ Encoding nominal categorical variables in unsupervised learning problems while o
 
 #### Steps to get you started:
 
-1. Visit [jupyterhub.datahub.redhat.com](https://jupyterhub.datahub.redhat.com/)
-2. Login with your Kerberos ID
+1. Visit https://jupyterhub-opf-jupyterhub.apps.cnv.massopen.cloud/hub/login
+2. Login with your google account
 3. In this Spawn screen, select s2i-categorical-encoding-notebook:latest, in the memory size select Large ( Memory: 8Gi, CPU 2)
-4. Once your server starts, go into the directory named categorical-encoding-yyyy-mm-dd-hh-mm/demo
+4. Once your server starts, go into the directory named notebooks-yyyy-mm-dd-hh-mm/demo
```
Comment (tumido) [inline docs/blog/blog.md]: This should be reverted. As per https://github.com/operate-first/odh-moc-support/issues/24#issuecomment-735492292 after an image rebuild https://github.com/aicoe-aiops/categorical-encoding/issues/15, we should have the project name included in the folder name. Please check it after the build if it's indeed true.

**Review by tumido — CHANGES_REQUESTED** [review]
:exploding_head: some changes became obsolete, even before merged :smile:

File: docs/blog/blog.md (hunk 1 above)
Comment (Shreyanand) [reply docs/blog/blog.md]: I still see _notebooks_ but it's a matter of time I guess. I'll revert it!

Comment (tumido) [reply docs/blog/blog.md]: yeah, the sesheta is not behaving today https://github.com/aicoe-aiops/categorical-encoding/issues/15 it'll be fixed soon. :slightly_smiling_face:

**Review by tumido — APPROVED** [review]

**Pull request merged**